bin/rainforest_cd

```diff
@@ -3,18 +3,14 @@
 # Die on error
 set -e
 
-gem install rainforest-cli
-
-echo $PATH
-ruby --version
-gem list
-rvm gemset list
-
 # Run all Rainforest tests. RAINFOREST_TOKEN is an environment variable
 # that needs to be configued in CircleCI's UI. It represents the API token
 # of your Rainforest account. You can find it in your account's settings.
 rainforest run all --token $RAINFOREST_TOKEN --fg --fail-fast
```

Comment:
Do we still want `--fail-fast` as the default example? The Orb doesn't use it on the Circle side of things.